src/components/PluginGovernanceStatus.jsx

```diff
@@ -8,38 +8,25 @@ function PluginGovernanceStatus({plugin}) {
     return plugin.labels.map((id) => {
         if (id === 'adopt-this-plugin') {
             return (
-                <div className="plugin-governance-notice">
-                    <table>
-                        <tr>
-                            <td className="badge-box"><span className="badge active warning"/></td>
-                            <td className="plugin-governance-notice-text">
-                                <b>This plugin is up for adoption!</b>
-                                {' We are looking for new maintainers. Visit our '}
-                                <a href="https://jenkins.io/doc/developer/plugin-governance/adopt-a-plugin/">Adopt a Plugin</a>
-                                {' initiative for more information.'}
-                            </td>
-                        </tr>
-                    </table>
+                <div className="alert alert-warning alert-with-icon">
+                    <span className="alert-icon warning" aria-label="Warning"/>
+                    <b>This plugin is up for adoption!</b>
+                    {' We are looking for new maintainers. Visit our '}
+                    <a href="https://jenkins.io/doc/developer/plugin-governance/adopt-a-plugin/">Adopt a Plugin</a>
+                    {' initiative for more information.'}
                 </div>
             );
         } else if (id === 'deprecated') {
             return (
-                <div className="plugin-governance-notice">
-                    <table>
-                        <tr>
-                            <td className="badge-box"><span className="badge active warning"/></td>
-                            <td className="plugin-governance-notice-text">
-                                <b>Deprecated:</b>
-                                {'This plugin has been marked as deprecated. '}
-                                {'In general, this means that this plugin is either obsolete, no longer being developed, or may no longer work. '}
-                                {'See the documentation for further information about the cause for the deprecation, and suggestions on how to proceed. '}
-                                {'The deprecation process is also documented '}
-                                <a href="https://jenkins.io/doc/developer/plugin-governance/deprecating-or-removing-plugin">here</a>
-                                {'.'}
-
-                            </td>
-                        </tr>
-                    </table>
+                <div className="alert alert-warning alert-with-icon">
+                    <span className="alert-icon warning" aria-label="Warning"/>
+                    <b>Deprecated:</b>
+                    {'This plugin has been marked as deprecated. '}
```

Comment:
FYI, you can use a backtick and make this one large blob, like java's 3 quotes. At some point I want to find out why eslint messes with strings

(I know you didn't make these strings just reformatted, but wanted to share)